content/id/docs/concepts/scheduling/scheduler-perf-tuning.md

```diff
@@ -0,0 +1,161 @@
+sebuah Pod disebut sebagai Node-Node layak (_feasible_). Penjadwal mencari Node-Node
+layak untuk sebuah Pod dan kemudian menjalankan fungsi-fungsi untuk menghitung
+penilaian (_score_) dari Node-Node tersebut, memilih sebuah Node dengan penilaian tertinggi di antara
+Node-Node layak lainnya, dimana Pod akan dijalankan. Penjadwal kemudian memberitahu
+peladen (_server_) API soal keputusan ini melalui sebuah proses yang disebut _Binding_.
+
+Laman ini menjelaskan optimasi penyetelan (_tuning_) performa yang relevan
+untuk klaster Kubernetes berskala besar.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+Pada klaster berskala besar, kamu bisa menyetel perilaku penjadwal
+untuk menyeimbangkan keluaran penjadwalan antara _latency_ (seberapa cepat Pod-Pod baru ditempatkan)
+dan akurasi (seberapa akurat penjadwal membuat keputusan penjadwalan yang tepat).
+
+Kamu bisa mengkonfigurasi setelan ini melalui pengaturan `percentageOfNodesToScore` pada kube-scheduler.
+Pengaturan KubeSchedulerConfiguration ini menentukan sebuah _threshold_ untuk
+penjadwalan Node-Node di dalam klaster kamu.
+
+### Pengaturan _threshold_
+
+Opsi `percentageOfNodesToScore` menerima semua angka numerik antara 0 dan 100.
+Angka 0 adalah angka khusus yang menandakan bahwa kube-scheduler harus menggunakan
+nilai _default_.
+Jika kamu mengatur `percentageOfNodesToScore` dengan angka di atas 100, kube-scheduler
+akan membulatkan ke bawah menjadi 100.
+
+Untuk mengubah angka, sunting berkas konfigurasi kube-scheduler (berkas `/etc/kubernetes/config/kube-scheduler.yaml`),
+lalu _restart_ ulang penjadwal.
+
+Setelah kamu selesai menyunting, jalankan perintah
+```bash
+kubectl get componentstatuses
+```
+untuk verifikasi komponen kube-scheduler berjalan dengan baik (_healthy_). Keluarannya kira-kira seperti ini:
+```
+NAME                 STATUS    MESSAGE             ERROR
+controller-manager   Healthy   ok
+scheduler            Healthy   ok
+...
+```
+
+## _Threshold_ penilaian Node {#persentase-penilaian-node}
+
+Untuk meningkatan performa penjadwalan, kube-scheduler dapat berhenti mencari
+Node-Node yang layak saat sudah berhasil menemukannya. Pada klaster berskala besar,
+ini menghemat waktu dibandingkan dengan pendekatan awam yang mengecek setiap Node.
+
+Kamu bisa mengatur _threshold_ untuk menentukan berapa banyak jumlah Node minimal yang dibutuhkan, sebagai
+persentase bagian dari seluruh Node di dalam klaster kamu. kube-scheduler akan mengubah ini menjadi
+bilangan bulat berisi jumlah Node. Saat penjadwalan, jika kube-scheduler mengidentifikasi
+Node-Node layak yang diperlukan telah melewati jumlah persentase yang diatur, maka kube-scheduler
+akan berhenti mencari Node-Node layak dan lanjut ke [fase penilaian] (/docs/concepts/scheduling/kube-scheduler/#kube-scheduler-implementation).
+
+[Bagaimana penjadwal mengecek Node](#bagaimana-penjadwal-mengecek-node) menjelaskan proses ini secara detail.
+
+### _Threshold default_
```

Comment:
```suggestion
### Ambang Batas Bawaan
```

Reply:
Fixed in https://github.com/kubernetes/website/pull/20260/commits/3cc163e65705127d1fcf16feba108179f9bdacc2